### PR TITLE
Add ibis example

### PR DESCRIPTION
**Pull request by cpcloud**
This PR adds an Ibis example alongside the other Python examples. No pressure to accept, just thought I would put up a PR in case folks are amenable to it!

### CODE REVIEW

File: _includes/landing-page/python/ibis-integration.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import ibis
+from ibis import _
+
+t = ibis.memtable({
+    'station': ['Delft', 'Delft', 'Gouda', 'Gouda'],
+    'day': ['Mon', 'Tue', 'Mon', 'Tue'],
+    'num_services' : [22, 20, 27, 25]})
+
+# execute a query on duckdb and return a dataframe
+data = t.group_by("station").agg(_.num_services.sum()).to_pandas()
```
Comment (lostmygithubaccount) [inline _includes/landing-page/python/ibis-integration.py:11]: might we worth adding a UDF example too? perhaps just the `count_vowels`?

Comment (cpcloud) [reply _includes/landing-page/python/ibis-integration.py:11]: Happy to add, but would like to wait for feedback before adding more!

**Comment by szarnyasg** [comment]
Hi @cpcloud, thanks for the PR! Indeed (and unfortunately), we cannot merge this – we would like to keep our landing site lightweight and we also decided against including other libraries such as `numpy`.

**Comment by cpcloud** [comment]
Ah, interesting!

numpy is a pandas dependency, so it's already in there FWIW. I guess it's something else then?

**Comment by szarnyasg** [comment]
It's just a matter of having limited real estate on the landing page. We can only expect a certain amount of attention from users clicking through various examples for the languages displayed there.

**Comment by cpcloud** [comment]
For sure! Thanks for the feedback!

**Pull request closed without merging**